runtime/registry.go

```diff
@@ -333,18 +347,18 @@ func (r *registryCache) restartController(iwc *instanceWithController) {
 			// Before starting the controller, sync the repo.
 			// This is necessary for resources (usually sources or models) that reference files in the repo,
 			// and may be triggered before the project parser is triggered and syncs the repo.
-			r.logger.Info("syncing repo", zap.String("instance_id", iwc.instance.ID))
+			iwc.logger.Debug("syncing repo", zap.String("instance_id", iwc.instance.ID))
```

Comment:
I think `zap.String("instance_id", iwc.instance.ID)` is redundant when using `iwc.logger` since it's already passed in `With`. This applies a few places in this function.